docs/user-documentation/users.md

```diff
@@ -0,0 +1,36 @@
+# Managing Users
+
+## How to Add a User
+1. Log in to Drupal under admin credentials.
+1. Click the **Manage** tab from the top toolbar.
+1. Select the **People** tab from the resulting menu to bring up the *People* page.
+![Alt text](../assets/users_people.png "People page")
+1. Click the blue **Add user** button to bring up the *Add user* page.
+1. Enter all required fields, as indicated by red asterisks, in the *Add user* page.
+1. In the *Roles* section, click on the roles that the new user will need.
```

Comment:
Suggested addition:
In the *Contact Settings* section.  Note: the Personal contact form is selected by default.